downstream/modules/platform/ref-schedule-jobs-worker-nodes.adoc

```diff
@@ -0,0 +1,21 @@
+[id="ref-schedule-jobs-worker-nodes"]
+
+= Scheduling jobs on the worker nodes
+
+In short, both controller and kubernetes play a role in scheduling a job.  
+
+When a job is launched, its dependencies are fulfilled, meaning any project updates or inventory updates are launched by Controller.s" settings respected, etc. 
```

Comment:
I don't understand the end of this sentence.  Should it be Controller.s" settings resepcted. It doesn't seem to make sense.

Reply:
```suggestion
When a job is launched, its dependencies are fulfilled, meaning any project updates or inventory updates are launched as indicated by settings on each resource (project update on launch, etc.)
```